charts/internal/seed-controlplane/charts/aws-load-balancer-controller/values.yaml

```diff
@@ -0,0 +1,332 @@
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+  # Automount API credentials for a Service Account.
+  automountServiceAccountToken: false
+  # List of image pull secrets to add to the Service Account.
+  imagePullSecrets:
+  # - name: docker
+
+rbac:
+  # Specifies whether rbac resources should be created
+  create: true
+
+podSecurityContext:
+  fsGroup: 65534
+
+securityContext:
+  # capabilities:
+  #   drop:
+  #   - ALL
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  allowPrivilegeEscalation: false
+
+# Time period for the controller pod to do a graceful shutdown
+terminationGracePeriodSeconds: 10
+
+resources:
+  limits:
+    memory: 512Mi
+  requests:
+    memory: 128Mi
+
+# priorityClassName specifies the PriorityClass to indicate the importance of controller pods
+# ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass
+priorityClassName: gardener-system-300
+
+nodeSelector: {}
+
+tolerations: []
+
+# affinity specifies a custom affinity for the controller pods
+affinity: {}
+
+# configureDefaultAffinity specifies whether to configure a default affinity for the controller pods to prevent
+# co-location on the same node. This will get ignored if you specify a custom affinity configuration.
+configureDefaultAffinity: true
+
+# topologySpreadConstraints is a stable feature of k8s v1.19 which provides the ability to
+# control how Pods are spread across your cluster among failure-domains such as regions, zones,
+# nodes, and other user-defined topology domains.
+#
+# more details here: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
+topologySpreadConstraints: {}
+
+updateStrategy: {}
+  # type: RollingUpdate
+  # rollingUpdate:
+  #   maxSurge: 1
+#   maxUnavailable: 1
+
+# deploymentAnnotations contains annotations for the controller deployment
+deploymentAnnotations: {}
+
+podAnnotations: {}
+
+podLabels:
+  gardener.cloud/role: controlplane
+  networking.gardener.cloud/to-runtime-apiserver: allowed
+  networking.resources.gardener.cloud/to-kube-apiserver-tcp-443: allowed
+  networking.gardener.cloud/to-dns: allowed
+  networking.gardener.cloud/to-public-networks: allowed
+  maintenance.gardener.cloud/restart: "true"
+
+# additionalLabels -- Labels to add to each object of the chart.
+additionalLabels:
+
+# Enable cert-manager
+enableCertManager: false
+
+# The name of the Kubernetes cluster. A non-empty value is required
+clusterName:
+
+# cluster contains configurations specific to the kubernetes cluster
+cluster:
+  # Cluster DNS domain (required for requesting TLS certificates)
+  dnsDomain: cluster.local
+
+# The ingress class this controller will satisfy. If not specified, controller will match all
+# ingresses without ingress class annotation and ingresses of type alb
+ingressClass: alb
```

Comment:
If we consider, not creating the ingress class/params ourselves, then we should probably expose the name and have it defaulted to `alb` if empty.